src/main/java/frc/robot/utils/DeltaAngleSpeedCalcUtil.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants;
+
+public class DeltaAngleSpeedCalcUtil {
+  private static final double SPEAKER_HEIGHT_INCHES = Units.inchesToMeters(77.9528);
```

Comment:
unless I'm misinterpreting, wouldn't inchesToMeters() return a value in meters? so shouldn't SPEAKER_HEIGHT_INCHES actually be SPEAKER_HEIGHT_METERS?

Reply:
Great point thanks 
I will be making the google doc much more easier to read but here's the link https://docs.google.com/document/d/1rIMmzDA1EMH7NQexIMrNYAD-_KFLe9cL4zPLXdl0lek/edit?usp=sharing
its kinda long so sorry about that

the issue is the diagrams kinda take a little to make because i have to replicate it from my own notes and sometimes my notes are illegible..